### ISSUE TITLE
[Question] Gradle reports are are broken 

### ISSUE THREAD

**Issue by srinivasbudh** [serenity-bdd/serenity-cucumber4-starter#2]
Hi Team,

I have used Serenity Reporting with maven in the past, I was trying to implement a new project using gradle but requirements are not loaded properly using gradle.

I tired running this project as well with maven and gradle. When run with maven reports were generated properly, however when I run this project using gradle reports are not displayed as expect (Features and Capabilities tabs are not loaded, also Requirements tab is displayed empty)

![image](https://user-images.githubusercontent.com/24290981/55032038-535fd900-5010-11e9-8066-1b7cefbe5c10.png)

Not sure if this gradle issue, Can someone please suggest on this

**Comment by wakaleo** [comment]
There is a known issue with Gradle and Java 11

**Comment by srinivasbudh** [comment]
Thanks for letting me know, does this mean if I install old version of Java, reports will be generated properly??

**Comment by wakaleo** [comment]
Yes, or an older version of Gradle (the problem seems to come from Gradle).

**Comment by srinivasbudh** [comment]
I have lowered my java version to java 1.8 and gradle to 4.9 but I still have this issue
